collections/traitattr/occurattributes.php

```diff
@@ -1,36 +1,38 @@
 <?php
 include_once('../../config/symbini.php');
-include_once($SERVER_ROOT.'/classes/OccurrenceAttributes.php');
-header("Content-Type: text/html; charset=".$CHARSET);
+include_once($SERVER_ROOT . '/classes/OccurrenceAttributes.php');
 
-if(!$SYMB_UID) header('Location: '.$CLIENT_ROOT.'/profile/index.php?refurl=../collections/traitattr/occurattributes.php?'.htmlspecialchars($_SERVER['QUERY_STRING'], ENT_QUOTES));
+if ($LANG_TAG != 'en' && file_exists($SERVER_ROOT . '/content/lang/collections/traitattr/occurattributes.' . $LANG_TAG . '.php')) include_once($SERVER_ROOT . '/content/lang/collections/traitattr/occurattributes.' . $LANG_TAG . '.php');
+else include_once($SERVER_ROOT . '/content/lang/collections/traitattr/occurattributes.en.php');
+
+header("Content-Type: text/html; charset=" . $CHARSET);
+
+if (!$SYMB_UID) header('Location: ' . $CLIENT_ROOT . '/profile/index.php?refurl=../collections/traitattr/occurattributes.php?' . htmlspecialchars($_SERVER['QUERY_STRING'], ENT_QUOTES));
 
 $collid = $_REQUEST['collid'];
-$submitForm = array_key_exists('submitform',$_POST)?$_POST['submitform']:'';
-$mode = array_key_exists('mode',$_REQUEST)?$_REQUEST['mode']:1;
-$traitID = array_key_exists('traitid',$_REQUEST)?$_REQUEST['traitid']:'';
-$paneX = array_key_exists('panex',$_POST)?$_POST['panex']:'575';
-$paneY = array_key_exists('paney',$_POST)?$_POST['paney']:'550';
-$imgRes = array_key_exists('imgres',$_POST)?$_POST['imgres']:'med';
+$submitForm = array_key_exists('submitform', $_POST) ? $_POST['submitform'] : '';
```

Comment:
Did some auto-formatting happen here, or did you add all these spaces? (to discuss)